src/trading/completeSets.se

```diff
@@ -105,7 +105,7 @@ def sellCompleteSets(sender: address, market: address, fxpAmount: int256):
     if reportingFee:
         require(denominationToken.transferFrom(market, reportingWindow, reportingFee))
     require(denominationToken.transferFrom(market, sender, sellerPayout))
-
+    
```

Comment:
Style: no spaces on empty lines.  I _believe_ if you use an editor that supports .editorconfig this will happen for you with the `trim_trailing_whitespace` setting.